LayoutTests/imported/w3c/web-platform-tests/webvtt/api/VTTCue/align.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!doctype html><!-- webkit-test-runner [ captionDisplayMode=automatic ] -->
```

Comment:
Would this not get removed the next time we run an import?

Reply:
This comment is preserved the next time it is imported. `test_importer.py` checks if the existing test has a `webkit-test-runner` comment and inserts it back in the new test file. See https://github.com/WebKit/WebKit/commit/24470f7b90f2685ea50b0e268d212d34f5b46d0c for details.